scripts/binpadder.sh

```diff
@@ -22,15 +22,12 @@ echo "UNIT: $unit"
 case "$unit" in
 	b | B | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 0)
 		final_size=$2
-		break
```

Comment:
Looks like mis-writing by the one who combined C syntax.

Reply:
Yeah, lol.